Makefile

```diff
@@ -3,11 +3,16 @@
 SUBDIR=	linuxkpi	\
 	lindebugfs	\
 	drm		\
-	i915		\
+	${_vmwgfx}	\
+	${_i915}	\
 	amd		\
 	radeon		\
 	${_staging}	
 
+.if ${MACHINE_CPUARCH} == "amd64" || ${MACHINE_CPUARCH} == "i386"
+_i915 =		i915 
+_vmwgfx =	vmwgfx
```

Comment:
Ah, I need to remove vmwgfx from the powerpc64 part of the port update. Good to know.